Lib/test/test_doctest/test_doctest.py

```diff
@@ -2418,6 +2428,18 @@ def test_DocFileSuite():
          Traceback (most recent call last):
          ValueError: Package may only be specified for module-relative paths.
 
+       If all examples in a file are skipped, unittest will report it as a
+       skipped test:
+
+         >>> suite = doctest.DocFileSuite('test_doctest.txt',
+         ...                              'test_doctest4.txt',
+         ...                              'test_doctest_skip.txt')
```

Comment:
Question: why do we add `test_doctest.txt` and `test_doctest4.txt` to the `DocFileSuite` here?

Reply:
It makes sure that skips can coexist with other tests, by making a suite containing one of each option: pass, fail and skip.